components/common/Buttons.tsx

```diff
@@ -12,36 +13,37 @@ import {
 type ButtonProps = {
   title: string
   style?: StyleProp<ViewStyle>
+  textStyle?: StyleProp<TextStyle>
```

Comment:
nitpick - I think it would be better to make a new variant of this component rather than making textStyle completely customizable, so people don't go too crazy with the button styling.